conu/apidefs/metadata.py

```diff
@@ -47,8 +47,8 @@ def __init__(self, name=None, identifier=None, labels=None, command=None,
         :param env_variables: dict, {name: value}
         :param image: Image, reference to Image instance
         :param exposed_ports: list, list of exposed ports
-        :param port_mappings: dict, dictionary of port mappings {"host_port": "container_port"}, example:
-            - {"8080":"80"} map port 80 in the container to port 8080 on the host
+        :param port_mappings: dict, dictionary of port mappings {"host_port": ["container_port"]}, example:
```

Comment:
Hm, this doesn't make much sense to me. How could one external port be mapped to two internal?